tests/unit/hmrc/mtdFraudPrevention.test.js

```diff
@@ -35,6 +38,7 @@ describe("FraudPreventionHeaders", () => {
       height: 1021,
       colorDepth: 17,
     });
+    jest.spyOn(uuidGeneratorHelper, "getDeviceId").mockReturnValue("af33f46c-085d-4050-be2a-98b72820c572");
```

Comment:
Same again, this test class is an integration test, i.e. it should not matter if we modify helper files, restructure our project. Mock external dependencies (`uuid` in this case) and test the end to end solution. A good thought experiment here to know if you are doing the right test: can you restructure your internal implementation without your test breaking? For example, if you were to just rename `getDeviceId` to be something else, your test will break... we don't want that! That is way too brittle.

Reply:
done

Reply:
removed dependency on getDeviceId